src/screens/EPub/views/EditINote/index.js

```diff
@@ -1,25 +1,77 @@
 import React, { useEffect, useState } from 'react';
 import _ from 'lodash'
-import { CTFragment, CTHeading, CTText } from 'layout';
+import cx from 'classnames';
+
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { CTFragment, CTHeading, CTText, altEl, useButtonStyles } from 'layout';
 import { connect } from 'dva'
+import Button from '@material-ui/core/Button';
 import { EPubNavigationProvider } from '../../components';
-import { epub as epubController} from '../../controllers';
+import { epub as epubController, generateEPubGuide} from '../../controllers';
 import INoteEditor from './INoteEditor';
+import Instruction from '../EditEPubStructure/Instruction';
+import EPubItemView from '../EditEPubStructure/EPubItemView';
+import QuickActionsEditNote from './QuickActionsEditNote';
+
 
 // import './index.scss';
 
 function EditINote ({epub: epubData, dispatch}) {
   const dispatchScroll = _.debounce((e) => dispatch({ type: 'epub/onScroll', payload: e }), 300)
   const onScroll = (e) => dispatchScroll(e.target)
-  
+
+  const [ePubItem, setEPubItem] = useState(null);
+  const [instExp, setInstExp] = useState(true);
+
+  const toggleInstExp = (e, newExpanded) => setInstExp(newExpanded);
+
+  useEffect(() => {
+    // show the user onboard guide if possible
+    setTimeout(() => {
+      const guide = generateEPubGuide();
+      guide.start();
+    }, 1000);
+  }, []);
+
+  useEffect(() => {
+    if (Boolean(ePubItem) && instExp) {
+      setInstExp(false);
+    }
+  }, [ePubItem]);
+  const itemViewElem = altEl(EPubItemView, Boolean(ePubItem), {
+    item: ePubItem, setEPubItem
+  });
+  const btnStyles = useButtonStyles();
+  const btnClasses = cx(btnStyles.tealLink, 'justify-content-start');
```

Comment:
I think most of this is unused functionality. We are no longer displaying the instruction and guide to the user. Delete all that does not pertain to the split menu.